bench.py

```diff
@@ -86,6 +86,14 @@ def add_benchmark_args(parser: argparse.ArgumentParser):
         action=argparse.BooleanOptionalAction,
         help="drop tables before starting",
     )
+    parser.add_argument(
```

Comment:
Can we achieve the desired auto-scaling by just running queries? We already have a mechanism for that, we just need to expose it in the cli arguments

Reply:
Ideally we don't want to create a bunch of tables just to warm up the cluster

Reply:
Change to scale up queries only, expose the warmup queries parameter which default to num of queries and can be specified by users